cmd/main.go

```diff
@@ -18,11 +18,11 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
 
+	crdv1alpha1 "github.com/vmware-tanzu/nsx-operator/pkg/apis/crd.nsx.vmware.com/v1alpha1"
```

Comment:
It uses the whole domain as the package name? I think it's uncommon, why doesn't it just name the package as "crd"?

In my mind the new group can be "crd" and the old can be "legacy".

Reply:
Yes, github.com/vmware-tanzu/nsx-operator/pkg/apis is the module name, and it contains `nsx.vmware.com` dir which is named as discussed, now we add the new name `crd.nsx.vmware.com`, that is better to be distinguished.

Reply:
Can you share where it was discussed for me to understand why it's made so? It's neither common in K8s nor Go to have a domain as a package name.

Reply:
@dantingl @timdengyun What's your opinion? How about arranging it in the next PR? It is not urgent.

Reply:
Please create a github issue to address it in a following PR.

Reply:
ok